src/aeb_fate_asm_scan.template

```diff
@@ -102,7 +109,17 @@ parse_acc("a" ++ N) -> list_to_integer(N).
 
 
 parse_hash("#" ++ Chars) ->
-    base58_to_address(Chars).
+    base64:decode(Chars).
+
+parse_object([_|Chars]) ->
```

Comment:
Shouldn't you match on the first character as well? Is it really anything?

Reply:
It is either @ or $ but the scanner makes sure of that, matching on that with a when-clause ( C == $$ or c == $@ ) could be done but it would be a bit redundant.